examples/todomvc/store/mutations.js

```diff
@@ -15,7 +15,6 @@ export const mutations = {
   },
 
   editTodo (state, { todo, text = todo.text, done = todo.done }) {
-    todo.text = text
-    todo.done = done
+    state.todos.splice(state.todos.indexOf(todo), 1, { ...todo, text, done })
```

Comment:
Can we refactor it for readability? Having everything on one line looks a bit too packed.

```suggestion
    const index = state.todos.indexOf(todo)
    state.todos.splice(index, 1, { 
      ...todo, 
      text, 
      done 
    })
```

Reply:
Ah of course 👍

Reply:
Done!